hack/images/dns/kube2sky/kube2sky.go

```diff
@@ -0,0 +1,292 @@
+/*
```

Comment:
Ok, fair enough.

> On May 11, 2015, at 06:39, James DeFelice <notifications@github.com> wrote:
> 
> In hack/images/dns/kube2sky/kube2sky.go:
> 
> > @@ -0,0 +1,292 @@
> > +/*
> Actually, it's going to make rebasing a little more error prone. Because godep is not the best tool. And we're going to have to rebase at least one more time as part of the upstream effort. I'd rather just stick this here for now and clean it up later since it'll disappear once we upstream.
> 
> —
> Reply to this email directly or view it on GitHub.
>